### ISSUE TITLE
Add support for mssql

### ISSUE THREAD

**Issue by f8dca** [lovasoa/SQLpage#2]
It would be great if SQLPage would support Microsoft SQL server as a back end.

```
thread 'main' panicked at 'Invalid database URL: Configuration("database URL has the scheme of a MSSQL database but the `mssql` feature is not enabled")', src\webserver\database\mod.rs:230:34

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by lovasoa** [comment]
That would be great indeed ! Would you be interested in helping implement and maintain mssql support in SQLPage ?

**Comment by f8dca** [comment]
I can test it for sure. I do not use rust so I’m not sure how I can contribute to the  actual coding part of the project GaborOn Jun 6, 2023, at 7:10 PM, Ophir LOJKINE ***@***.***> wrote:﻿
That would be great indeed ! Would you be interested in helping implement and maintain mssql support in SQLPage ?

**Comment by f8dca** [comment]
I see that according to the changelog v0.9.0 now has support for MS SQL Server databases but it is not (yet) available on the release page. When do you think this will be public for testing ?

Thanks a lot

**Comment by lovasoa** [comment]
Hey ! I'm in the process of ironing out the last little things ! It'll be today or tomorrow :slightly_smiling_face:

**Comment by lovasoa** [comment]
I just released v0.9.0, with experimental support for SQL Server :sunglasses: : https://github.com/lovasoa/SQLpage/releases/tag/v0.9.0

If you encounter any problem while using it, please open new issues here, I'll be happy to help :)
